src/components/flat-table/flat-table-checkbox/flat-table-chackbox.component.d.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
```

Comment:
This file is named `flat-table-chAckbox` :)

Reply:
haha great spot, will fix it thanks